build/Uno.UI.Build.csproj

```diff
@@ -56,7 +56,8 @@
 		<ItemGroup>
 			<_PackageToUpdate Include="Uno.UI" />
 			<_PackageToUpdate Include="Uno.UI.RemoteControl" />
-			<_PackageToUpdate Include="Uno.UI.RemoteControl.Messaging" />
+			<_PackageToUpdate Include="Uno.UI.DevServer" />
+			<_PackageToUpdate Include="Uno.UI.DevServer.Messaging" />
```

Comment:
I don't see any usage of `_PackageToUpdate`.

Reply:
Indeed we can remove.